docs/site/Context.md

```diff
@@ -574,11 +574,14 @@ expects configuration. For example:
 
 The following APIs are available to enforce/leverage this convention:
 
-1. ctx.configure('servers.RestServer.server1') => Binding for the configuration
-2. ctx.getConfig('servers.RestServer.server1') => Get configuration
-3. `@config` to inject corresponding configuration
-4. `@config.getter` to inject a getter function for corresponding configuration
-5. `@config.view` to inject a `ContextView` for corresponding configuration
+1. `ctx.configure('servers.RestServer.server1')` => Binding for the
+   configuration
+2. `binding.configure(servers.RestServer.server1)` => Creates a accompanying
```

Comment:
I believe the syntax is slightly different?

```ts
Binding.configure('servers.RestServer.server1')
```

Reply:
Fixed.